.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/Users/matheus.rocha/Developer/portal/src/utils/volunteers.js":"1"},{"size":1401,"mtime":1630569472217}]
```

Comment:
O caminho pra tua máquina ficou aqui 😅
seria bom adicionar  também  checks pra quebra de linha

Reply:
Merda 😄 Esqueci de colocar isso no .gitignore

Reply:
Checks para quebra de linha?

Reply:
Isso. Tipo o end-of-line-fixer: https://pre-commit.com/hooks.html

Reply:
O Prettier já faz isso por padrão. Ele acabou não fazendo nesse arquivo porque ele não estava na lista de arquivos que prettier deveria rodar.